vic/drivers/python/vic_headers.py

```diff
@@ -1250,6 +1250,7 @@
     _Bool SIGNED;
     _Bool SUPPLIED;
     double multiplier;
+    char varname[2048];
```

Comment:
The python headers are "preprocessed".  So all the variables are expanded.  At some point, we'd like to create these at build time on the fly but that is a `someday` task at this point.